server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -86,7 +86,11 @@ export class TransformHandler {
                 containsUnsupportedViews
             )
         } catch (error) {
-            const errorMessage = (error as Error).message ?? 'Error in StartTransformation API call'
+            let errorMessage = (error as Error).message ?? 'Error in StartTransformation API call'
+            if (errorMessage.includes('Invalid transformation specification')) {
+                errorMessage =
```

Comment:
I think you need to change the field (error as Error).message ? do we popup this error in the info bar/message box?

Reply:
just log. The error message box would still show "Failed to start transformation job."